storage/reports/reports_test.go

```diff
@@ -0,0 +1,680 @@
+package reports
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/golang/mock/gomock"
+	"github.com/jinzhu/gorm"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/reports"
+	"github.com/iryonetwork/wwm/storage/reports/db/mock"
```

Comment:
could not import github.com/iryonetwork/wwm/storage/reports/db/mock (cannot find package "github.com/iryonetwork/wwm/storage/reports/db/mock" in any of:
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/storage/reports/db/mock (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/storage/reports/db/mock (from $GOROOT)
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/storage/reports/db/mock (from $GOPATH))